### PR TITLE
Corrected the license to be LogRhythm MIT and adjusted the README

### PR DESCRIPTION
**Pull request by KjellKod**
This pull request corrects the LogRhythm LICENSE and improves the Readme.

The README can best be read at: https://github.com/KjellKod/FileIO/blob/LogRhythm_OpenSource_Compliance/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
 
-A generic C++ file IO class
+Is a collection of file access and directory traversal utilities that was written to be a fast, easy to use and non-exception throwing library. 
+
+The major areas it deals with are:
+
+*_Directory  traversal and file access actions_*:  
+The _DirectoryReader_ and _FileSystemWalker_ are efficient for traversing directory structures and accessing files with ad-hoc actions on those files.  The code was written to be efficient, fast and easy to use. 
+
+Errors are handled through return types and embedded error messages with the result.   In an error-prone scenario the FileIO execution is significantly faster than than solutions that utilizes exceptions.
```
Comment (Schatzman) [inline README.md]: ...than solutions that __utilize__ exceptions.

Comment (Schatzman) [inline README.md]: Also, "than" is written twice.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+The major areas it deals with are:
+
+*_Directory  traversal and file access actions_*:  
+The _DirectoryReader_ and _FileSystemWalker_ are efficient for traversing directory structures and accessing files with ad-hoc actions on those files.  The code was written to be efficient, fast and easy to use. 
+
+Errors are handled through return types and embedded error messages with the result.   In an error-prone scenario the FileIO execution is significantly faster than than solutions that utilizes exceptions. 
+
+
+**File Access**  
+The file access code was written to avoid rewriting boiler plate code that is easy to get wrong. It covers areas such as file reading and writing, changing file ownership, deleting files and directories and moving of files. Many of these functionalities involve low-level C or C++ API that are awkward to use and prone to have bugs.
```
Comment (Schatzman) [inline README.md]: ...and prone to bugs. -OR- ...and prone to having bugs.

File: README.md (hunk 1 above)
Comment (john-gress) [inline README.md]: "with the result" - do you mean "with a Result class object. See Result.h"?

File: README.md (hunk 2 above)
Comment (john-gress) [inline README.md]: "C or C++ API" -> "C or C++ APIs"

File: README.md (hunk 1 above)
Comment (KjellKod) [inline README.md]: got it.

File: README.md (hunk 2 above)
Comment (KjellKod) [inline README.md]: changed `having`

File: README.md (hunk 1 above)
Comment (KjellKod) [inline README.md]: @john-gress, yes, but I didn't call it out specifically.  I will add a link to the Result object.

Comment (KjellKod) [inline README.md]: updated

File: README.md (hunk 2 above)
Comment (KjellKod) [inline README.md]: done

**Comment by Schatzman** [comment]
:+1:

**Comment by john-gress** [comment]
👍

**Pull request merged**